src/Workspaces/Core/Portable/Utilities/ForegroundThreadDataKind.cs

```diff
@@ -27,7 +29,7 @@ static ForegroundThreadDataInfo()
             s_fallbackForegroundThreadDataKind = CreateDefault();
         }
 
-        internal static ForegroundThreadDataKind CreateDefault()
+        internal static ForegroundThreadDataKind CreateDefault(ForegroundThreadDataKind defaultKind = Unknown)
```

Comment:
can we not make this optional.  I'd really like all callers to be clear about what they're passing in.